### DISCUSSION TITLE
Popular time

### DISCUSSION THREAD

**Discussion by gio19g** [omkarcloud/google-maps-scraper#71] [Q&A]
I would like to know if it is possible to extract the popular time of a place. Thanks

**Comment by Chetan11-dev** [comment]
Thank you for your suggestion. At this point of time, I'm not planning to add this feature. 

Most importantly because there hasn't been demand for it from the users.

However, if more users express interest in this feature in the future, I will certainly reconsider my decision.

Best regards,
Chetan

**Comment by gio19g** [comment]
Hello

thanks for your answer

In my opinion popular time is an important statistic data

**Comment by Chetan11-dev** [comment]
Absolutely will add soon

**Comment by Chetan11-dev** [comment]
Added, Please follow these Steps. 

⿡ Clone the Magic 🧙‍♀:
```
git clone https://github.com/omkarcloud/google-maps-scraper
cd google-maps-scraper
```

⿢ Install Dependencies 📦:
```
python -m pip install -r requirements.txt
```

⿣ Let the Rain of Google Map Leads Begin 😎:
```
python main.py 
```

And Run the following command to update packages:

```
python -m pip install botasaurus --upgrade
```
